### PR TITLE
Desktop Environments Slide Deck

### PR DESCRIPTION
**Pull request by jrtechs**
Added slides to the website

### CODE REVIEW

File: talks/_posts/2020-01-31-desktop-environments.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Sprint 2020 - Desktop Environments
```
Comment (Tjzabel) [inline talks/_posts/2020-01-31-desktop-environments.md]: **Spring** 2020

File: talks/_posts/2020-01-31-desktop-environments.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+author: Jeffery Russell
+layout: post
+slides: 2020-spring-w03-desktop-environments.pdf
+date: 2020-01-31
+---
+
+In this presentation Time and Jeffery went over everything you need to know about Linux Window Managers and Desktop Environments!
```
Comment (Tjzabel) [inline talks/_posts/2020-01-31-desktop-environments.md]: **Tim** and Jeffery

File: talks/_posts/2020-01-31-desktop-environments.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Resources
+
+- [Wikapedia](https://en.wikipedia.org/wiki/Desktop_environment)
```
Comment (Tjzabel) [inline talks/_posts/2020-01-31-desktop-environments.md]: **Wikipedia**

File: talks/_posts/2020-01-31-desktop-environments.md (hunk 2 above)
Comment (jrtechs) [reply talks/_posts/2020-01-31-desktop-environments.md]: lolol I'll push the changes tomorrow

**Comment by Tjzabel** [comment]
@jrtechs you're still missing the _Spring 2020_ spelling mistake in the `title: ` field

**Pull request merged**